patterns/2-structured/support-as-a-group.md

```diff
@@ -0,0 +1,86 @@
+## Title
+
+Support as a Group
+
+## Patlet
+
+What happens when the team or individual that was supporting an InnerSource project is no longer able to do so?
+Form a group of interested individuals from anywhere in the company to keep the project going.
+
+## Problem
+
+* A popular InnerSource project is orphaned with no one to support it.
+* There isn't an obvious home for it to land.
+
+## Story
+
+A library of UI widgets is used by over 50 projects throughout the company.
+The funding for the team that owns the library runs out and the team disbands.
+At first, no one notices, but after a while whenever someone asks "who owns it" there isn't an answer.
+What will happen next?
+Will new teams shy away from using it?
+Will the project stagnate and linger until its users eventually are forced to move on to something else?
+What a shame if that were to happen to a perfectly good and useful project!
+
+## Context
+
+* Popular InnerSource project.
+* No one is actively supporting it.
+
+## Forces
+
+* No one is assigned by their day job to work on it.
+* Everyone is busy.
+* High cost to migrate off the project.
+
+## Solutions
+
+Call for interested volunteers from anywhere in the company to form a group of [Trusted Committer]s to support the project.
+You may need to reach out to specific individuals based on commit or usage history.
+It is important that there are enough so that the burden on each can be reasonably small.
+
+When forming, this group should identify or create:
+
+* A chat channel where questions and discussion about the project should happen.
+* Clear contributing instructions.
+* Link to both of the above in the project README.
+* A chat channel just for the [Trusted Committer]s.
+
+The group should do its best to manage these aspects of the project:
+
+* **Maintenance**.  If the project is flat-out broken for the standard use case, then fix it.
+Keep the project up-to-date as the dependencies and frameworks it uses continue to evolve.
+* **Onboarding**.  If someone has a question about how to use the project, make sure they get a reasonable answer.
+* **Updates**.  If someone wants to add new feature to the project, give them the design and technical support necessary for them to build it so that it both works for them and is a good addition to the project.
+Review incoming pull requests in a timely manner.
```

Comment:
I got the sense from this description that the Support Group won't typically implement requested features, will they? Instead they would try the help the requester to implement it themselves?

Maybe add something like this as well. i.e. what the Support Group won't do, or will unlikely be able to do, given their voluntary nature and "best effort" approach.

Reply:
`The group is not expected to implement any new functionality for others.`